specs/zabbix/zabbix.spec

```diff
@@ -182,9 +185,33 @@ Requires(postun):     %{__service}
 
 Conflicts:            zabbix-server-mysql
 
+################################################################################
+
 %description server-pgsql
 Zabbix server with PostgresSQL database support.
 
+%package java-gateway
+Summary:              Zabbix java gateway
+Group:                Applications/Internet
+
+%if 0%{?rhel} >= 7
+Requires:             java-headless >= 1.8.0
```

Comment:
Maybe is better to use java dependency for this package? We have different versions of OpenJDK packages in our repo and in this case we can't use them with this package.